pkg/networkmanager/interfaces.js

```diff
@@ -2902,8 +3223,30 @@ PageNetworkInterface.prototype = {
                                            configure_vlan_settings);
             }
 
-            return [render_master(),
-                render_autoconnect_row(),
+            function render_wifi_settings_row() {
+                var parts = [];
+                var rows = [];
+                var options = settings.wifi;
+                var security_options = settings.wifi_security;
+
+                if (!options)
+                    return null;
+
+                rows.push(_("SSID: " + (options.ssid ? atob(options.ssid) : "Not defined")));
```

Comment:
This translation would not work. Please do something like:
`cockpit.format(_("SSID: $0"), options.ssid ? atob(options.ssid) : _("Not defined"))`